### PR TITLE
Removed Tomcat 7 and Tomcat 8 support for 3.x.

### LINKED ISSUE #53: Remove Tomcat 7+8 support
Remove Tomcat 7 support for release 3.x.  See also discussion in https://github.com/gretty-gradle-plugin/gretty/pull/43#issuecomment-389379926 .

Also remove Tomcat 8.0 support, since this version was replaced by Tomcat 8.5.

### PR DESCRIPTION
**Pull request by javabrett**
This is the first of a few excisions to attempt to simplify the codebase for a 3.0 release and allow support for new features that collide with older container versions.

Tomcat 8.0 is EOL and was replaced by Tomcat 8.5 and 9.0.  Tomcat 7.x remains supported
but is well-supported in Gretty 2.x.

Fixed #53.

### CODE REVIEW

File: libs/gretty-runner-tomcat85/build.gradle [hunk 1]
```diff
@@ -11,7 +11,7 @@ dependencies {
   compile "org.apache.tomcat.embed:tomcat-embed-el:$tomcat85_version"
   compile "org.apache.tomcat.embed:tomcat-embed-jasper:$tomcat85_version"
   if (VersionNumber.parse(tomcat85_version) <= VersionNumber.parse('8.5.2'))
-    compile "org.apache.tomcat.embed:tomcat-embed-logging-log4j:$tomcat85_version"
```
Comment (boris-petrov) [inline libs/gretty-runner-tomcat85/build.gradle]: Why is this change needed? Why is it in this PR?

File: libs/gretty/src/main/groovy/org/akhikhl/gretty/ServletContainerConfig.groovy [hunk 2]
```diff
@@ -200,7 +148,7 @@ class ServletContainerConfig {
             force "org.apache.tomcat.embed:tomcat-embed-el:$tomcat85_version"
             force "org.apache.tomcat.embed:tomcat-embed-jasper:$tomcat85_version"
             if (VersionNumber.parse(tomcat85_version) <= VersionNumber.parse('8.5.2'))
-              force "org.apache.tomcat.embed:tomcat-embed-logging-log4j:$tomcat85_version"
```
Comment (boris-petrov) [inline libs/gretty/src/main/groovy/org/akhikhl/gretty/ServletContainerConfig.groovy]: Why is this change needed? Why is it in this PR?

**Comment by henrik242** [comment]
@boris-petrov Nah, let's keep it tomcat85. Tomcat distinguishes between 8.0 and 8.5, I guess we should too: http://tomcat.apache.org/

**Comment by javabrett** [comment]
> Perhaps we should rename tomcat85 to tomcat8?

-1 because Tomcat 8.5 is a distinct and incompatible release from Tomcat 8.0, and we want to be clear that we don't support 8.0 going forward, so 85 is clearer than 8.  I see where you are coming from though.

> You haven't removed libs/gretty-runner-tomcat8?

Good catch, thanks, removing left-over build file.

**Comment by boris-petrov** [comment]
Perhaps rename `*8plus` to `*85plus`?

**Comment by javabrett** [comment]
Thanks for the reviews.
